### PR TITLE
Add/update shopping files

### PR DESCRIPTION
**Pull request by rascalking**
Configured to allow shopping for Fall 2015 term.

### CODE REVIEW

File: js/shop.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
```
Comment (sapnamysore) [inline js/shop.js:3]: Have we verified if this the right term for colgsas? I see 69 for both hds and colgsas. 

In the previous setup,  I think we had discussed checking shop.js with blank allowed_terms and devops would populate the appropriate term during deployment for each semester. I am ok with a value but we may not know the correct term ids..

**Comment by sapnamysore** [comment]
:+1:

File: js/shop.js (hunk 1 above)
Comment (rascalking) [inline js/shop.js:3]: Yup, verified the term id with Jill, for both HDS and COLGSAS.  From talking to Colin in devops channel, it's my understanding that we should check in the final product to the school-specific repos they belong in, and the only processing devops is doing on deploy is concatenation/minification.

Comment (sapnamysore) [inline js/shop.js:3]: Sounds great!

Comment (cmurtaugh) [inline js/shop.js:3]: :+1:

**Pull request merged**